decred_wallet/extra_view/AccountsHeaderView/AccountsHeaderView.xib

```diff
@@ -114,7 +114,7 @@
                         </view>
                     </subviews>
                     <constraints>
-                        <constraint firstItem="ipD-e1-fPk" firstAttribute="width" secondItem="5HR-QN-iia" secondAttribute="width" multiplier="0.55" id="7L5-j2-BLx"/>
+                        <constraint firstItem="ipD-e1-fPk" firstAttribute="width" secondItem="5HR-QN-iia" secondAttribute="width" multiplier="0.51" id="7L5-j2-BLx"/>
```

Comment:
Could just use 0.5 so we know it's half of the parent width.

Reply:
Needed more space than 0.5

Reply:
Okay.